server/deployment/README.md

```diff
@@ -1,7 +1,47 @@
 = DevOps Scripts for configuring GCE, GKE, and K18S =
 
-Over time, this will host scripts to configure clouds to run
-the Arcs master, without needing to use the UI console.
+This directory contains scripts to bring up an instance of Arcs
+master on GCP. In the future, it will evolve to support other
+clouds such as AWS and Azure. The main multi-cloud portability
+comes from leveraging Kubernetes for orchestration and Helm for
+package and dependency management. 
+
+The cloud dependent portions of setup are in creating clusters and
+service accounts, setting IAM permissions, and configuring DNS
+providers. 
+
+
+== Installation Prerequisites ==
+Manual steps: Follow https://cloud.google.com/sdk/docs/quickstarts to install
+GCloud SDK.
+
+Install Docker (e.g for OSX https://docs.docker.com/docker-for-mac/install/)
+
+Also install Helm (https://docs.helm.sh/using_helm/#installing-helm)
+
+== Configuration Prerequisites ==
+
+Create a GCP account and enable billing. 
+
+Then run 'gcloud init'
+Then run 'gcloud components install kubectl'
+Then run 'gcloud components update'
+
+Create a domain on Google Domains registrar. Next, Setup a GCP CloudDNS zone for your
```

Comment:
Perhaps we should note that currently the configs are hard coded (?) for `skarabrae.org` as just an initial MVP, and that files X, Y, Z, etc. would need to be updated to reflect the alternate domain?

Could also be worth filing an issue to genericize the domain bits and referencing it here.